### DISCUSSION TITLE
Bridging issue from a WordPress site with federation already enabled

### DISCUSSION THREAD

**Discussion by ufalsalman** [snarfed/bridgy-fed#1642] [Q&A] [answered]
I read the documentation [here](https://fed.brid.gy/docs#web-get-started) about the "getting started" part for a website. My website is using WordPress and I have it running as a fediverse instance too for a while. In that case, theoretically I can't bridge my website using the service. Currently I have AP, WebFinger and Webmention plugin enabled. But if I disable the AP and WebFinger plugin first, can I actually bridge my site using Bridgy Fed?

**Comment by snarfed** [comment, marked as answer]
Hi! Yes, if you disable the ActivityPub plugin on your site so that it no longer serves Webfinger and AP, Bridgy Fed should work for you.

**Reply by ufalsalman** [reply]
Okay, thank you for the answer, Ryan!

**Reply by snarfed** [reply]
Welcome! Btw you'll need to re-enter your site into https://fed.brid.gy/#enter-web-site after you've disabled that plugin.

**Reply by rghedin** [reply]
By doing this (replacing AP plugin with Fed Bridgy), would I lose all followers I got when using the AP plugin?

**Reply by snarfed** [reply]
Yes, they'd be different AP accounts. We could support migrating your AP followers from one to the other once Bridgy Fed and the WP plugin both support the `Move` activity though! BF doesn't yet, https://github.com/snarfed/bridgy-fed/issues/330 , and I suspect the WP plugin doesn't either, but maybe eventually.

**Reply by rghedin** [reply]
WP plugin does support `Move` (kinda off) since earlier this month, in [v5.6.0](https://github.com/Automattic/wordpress-activitypub/releases/tag/5.6.0). Hope you guys can figure this out! :)

**Reply by snarfed** [reply]
Whoa, exciting, thanks for the heads up!
